apmpackage/apm/data_stream/app_logs/fields/ecs.yml

```diff
@@ -82,6 +82,12 @@
   name: event.severity
 - external: ecs
   name: event.action
+- external: ecs
+  name: event.type
+- external: ecs
+  name: event.category
+- external: ecs
+  name: event.kind
```

Comment:
Field changes should be accompanied by an enhancement entry in apmpackage/apm/changelog.yml

Reply:
Good catch! Updated :+1: